docs/.project/ALPHA_RELEASE.md

```diff
@@ -41,13 +42,12 @@ _To be done after code freeze and prior to the release day to prepare and build
     * [ ] Works on Mac
     * [ ] Works on Windows
 * [ ] trigger QA for fuzzy testing
-* [ ] write blog post for [blog.camunda.org](https://github.com/camunda/blog.camunda.org), __do not publish until released version is available for download on [camunda.org](https://camunda.org/release/camunda-modeler/)__
+* [ ] write blog post for [blog.camunda.org](https://camunda.com/blog/) according to [this process](https://confluence.camunda.com/pages/viewpage.action?spaceKey=MAR&title=Your+Content+on+the+Camunda+Blog) by copy and pasting [this template](https://docs.google.com/document/d/18aXWFBdbET97EZ_HO1DEVLVsYP-UQEabumSzQdeRbDw/edit?usp=sharing), writing a draft version of the blog post and sending it to Charley Mann and Christopher Rogers for review. Charley will coordinate the publishing.
```

Comment:
Instead of having one big task, we could split it up

```markdown
* [ ] write blog post for [blog.camunda.org](https://camunda.com/blog/) according to [this process](https://confluence.camunda.com/pages/viewpage.action?spaceKey=MAR&title=Your+Content+on+the+Camunda+Blog)
   * [ ] copy and paste [this template](https://docs.google.com/document/d/18aXWFBdbET97EZ_HO1DEVLVsYP-UQEabumSzQdeRbDw/edit?usp=sharing) to a new docs file
   * [ ] write a draft version of the blog post
   * [ ] send it to Charley Mann and Christopher Rogers for review

```

Reply:
:+1: